addons/hr/views/hr_employee_views.xml

```diff
@@ -105,9 +105,10 @@
                                             <field name="address_id"
                                                 context="{'show_address': 1}"
                                                 options='{"always_reload": True, "highlight_first_line": True}'/>
+                                            <field name="work_location"/>
                                         </group>
-                                        <group name="managers" string="Responsibles">
-                                            <field name="coach_id"/>
+                                        <group name="managers" string="Approvers">
```

Comment:
If you have only HR module installed, you will have a title "Approvers" with nothing bellow....

Reply:
Not really sure what's the best option here. 
If I remove the group, all the inheritance is broken. 
If I remove the string, the label won't be here. 
If I have to update all the children this piece of code, it becomes cumbersome and  really not code friendly. 

Is there maybe a way to hide if it contains no element ?

Reply:
They aren't way to hide if it contains no element. 
But you can add attribute invisible 
```suggestion
                                        <group name="managers" string="Approvers" invisible="1">
```

And when you add something in this group, you add also  
```
            <xpath expr="//group[@name='managers']" position="attributes">
                <attribute name="invisible">0</attribute>
            </xpath>
```

Reply:
I've just pushed this solution. I am really not found of it as it is too much for such tiny details.